modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/portlet/action/GetContentDashboardItemInfoMVCResourceCommand.java

```diff
@@ -142,6 +142,10 @@ protected void doServeResource(
 					"fetchSharingButtonURL",
 					_getFetchSharingButtonURL(
 						contentDashboardItem, httpServletRequest)
+				).put(
+					"fetchSharingContactsURL",
```

Comment:
hi @cgoncas !
you referenced [here](https://github.com/liferay-tango/liferay-portal/pull/2626#issuecomment-1195749318) `fetchSharingCollaboratorsURL` but I read here `fetchSharingContactsURL`. Which one should we use finally? As you said `fetchSharingCollaboratorsURL` makes more sense

Reply:
fetchSharingCollaboratorsURL should be the right one :), could you fix it :)

Reply:
Right, I'll fix it, thxs!